src/nbl/asset/utils/CIESProfileParser.cpp

```diff
@@ -150,12 +157,58 @@ bool CIESProfileParser::parse(CIESProfile& result)
         }
     }
 
-    const double factor = ballastFactor * candelaMultiplier;
-    for (int i = 0; i < hSize; i++)
-        for (int j = 0; j < vSize; j++)
-            result.setValue(i, j, factor * getDouble("intensity value truncated"));
-    
-    result.maxValue = *std::max_element(std::begin(result.data), std::end(result.data));
+    {
+        const double factor = ballastFactor * candelaMultiplier;
+        for (int i = 0; i < hSize; i++)
+            for (int j = 0; j < vSize; j++)
+                result.setCandelaValue(i, j, factor * getDouble("intensity value truncated"));
+    }
+
+    float totalEmissionIntegral = 0.0, nonZeroEmissionDomainSize = 0.0;
+    constexpr auto FULL_SOLID_ANGLE = 4.0f * core::PI<float>();
+
+    if (result.symmetry == CIESProfile::ISOTROPIC && result.vAngles.size() == 1) // not sure if any IES provides this kind of distribution but it should be possible to do so meaning fixed candela value for each direction
+    {
+        const auto candelaValue = result.getCandelaValue(0, 0);
+        constexpr auto solidAngle = FULL_SOLID_ANGLE; // spans whole sphere area
+
+        totalEmissionIntegral = candelaValue * solidAngle;
+        nonZeroEmissionDomainSize = solidAngle;
+    }
```

Comment:
not possible, why:
- if you look at your own validation this would only be posible for the array {90}
- your `CIESProfile::sample` would return non-zero distribution only for `vAngle=90`, meaning a 0 integral

TL;DR we should reject as invalid, emission profiles with `vSize<2` when loading

Reply:
we could either adjust them or reject as suggested, will take the second option